Cargo.toml

```diff
@@ -28,8 +30,10 @@ lazy_static.workspace = true
 libc.workspace = true
 modifier.workspace = true
 parking_lot.workspace = true
+regex = { workspace = true, optional = true }
 serde.workspace = true
 serde_json.workspace = true
+sha2.workspace = true
```

Comment:
can `sha2` also become optional and only enabled when `hide_ips` is turned on?

Reply:
Hmm, `hide_ips` was a replacement for having it enabled in debug mode only - because having it enabled in debug only seems to be better solution than that, and it seems that there is no way (i.e. I couldn't find one) to use a dependency only in release mode, so I guess both of them need to be compiled in all of the cases (As debug builds are not that optimized, it might not be a big issue)

Reply:
Got rid of this feature